jupyterhub/tests/conftest.py

```diff
@@ -43,73 +46,85 @@ def db():
 
 @fixture
 def io_loop():
-    """Get the current IOLoop"""
+    """Create a Tornado IOLoop and make it current."""
     loop = ioloop.IOLoop()
     loop.make_current()
     return loop
 
 
 @fixture(scope='module')
 def app(request):
+    """Start a Hub"""
+    # TODO - Q: what tracks the MockHub instance so it can be cleared later.
```

Comment:
The `.instance` method itself does this. It's tracked on the class. The instance is cleared with `clear_instance` in the finalizer immediately below.